lib/api/graphql/mutation/create_response_mutation_input.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'create_response_mutation_input.g.dart';
+
+@JsonSerializable()
+class CreateResponseMutationInput {
```

Comment:
same here `CreateSurveyResponseMutationInput` or shorter `CreateSurveyResponseInput` 🙏

Reply:
this follows the convention from the graphQL endpoint 🤔  I think I'd update the UseCase name but not this one 
![image](https://user-images.githubusercontent.com/13435717/126098378-f0e7f2c9-9943-46d8-ad54-e0dd6855db5c.png)